src/nrnmpi/mpispike.cpp

```diff
@@ -103,7 +103,7 @@ int nrnmpi_spike_exchange(int* ovfl,
                           NRNMPI_Spike* spikeout_,
                           NRNMPI_Spike** spikein_,
                           int* icapacity_) {
-    int i, n, novfl, n1;
```

Comment:
I realise it is tedious but I think one has to search for use in the entire function to include commented out (debugging?) statements and use in ```#if ... #else ...#endif``` conditionals. In the old K&R c it was necessary to have all declaration at the beginning, but now that it is c++, perhaps it will be more convenient to declare locally at the point of use. E.g.:
```
        int n1 = spbufin_[i].nspike;
```

Reply:
Fixed